test/test_aggregation.sh

```diff
@@ -33,7 +33,7 @@ if [ ! -f $BIG_SRS ]; then
 fi
```

Comment:
Can we remove  test_aggregation.sh. test_aggregation_verifier, and merge test_fibonacchi_verifier and test_poseidon_verifier ?